### DISCUSSION TITLE
Rekomendasi Build PC Harga Pelajar

### DISCUSSION THREAD

**Discussion by pomoheri** [deaafrizal/forum#19] [All About Hardware ( PC, Laptop etc...)]
Bang dea, beri formula pc yg ramah di kantong pelajar dengan budget 4 - 5 jt, dengan kebutuhan untuk coding dan sedikit game" ringan bang. Thanks bang

**Comment by deaafrizal** [comment]
sisanya bisa dibeliin aksesoris2 penunjang 👌

1. AMD Ryzen 5 5600G
2. Motherboard asus / asrock / msi A320M
3. PSU FSP hexa 400 / 500W
4. RAM 16GB apacer nox ddr4
5. SSD nvme m.2 adata xpg6000 256gb

**Comment by ghost** [comment]
•ASUS Vivobook 14 A1404VA-VIPS351 Ci3-1315U 8GB 512GB
•LENOVO IdeaPad 3 14ITL6-82H701MXID i5-1155G7 8GB 512GB SSD Intel Iris Xe Graphics 
•Macbook air m1 8gb 256gb

klok untuk ngoding lebih bagus dan awet yang mana kak?

**Reply by kenzieram** [reply]
macbook 256 kurang

**Reply by lahnan** [reply]
bisa di upgrade tenang
